hack/run-functests.sh

```diff
@@ -32,7 +36,7 @@ mkdir -p "$TESTS_REPORTS_PATH"
 if [ "$CNF_TESTS_IMAGE" != "" ]; then
```

Comment:
CNF_TESTS_IMAGE can be derived from TEST_POD_IMAGES_REGISTRY/TEST_POD_CNF_TEST_IMAGE
same goes for DPDK_TESTS_IMAGE

Reply:
I don't get the `DPDK_TESTS_IMAGE` part

Reply:
Also, CNF_TESTS_IMAGE is used as driver to choose the containerized version, whereas overriding the other two is optional. I can remove that variable, but I would need to reintroduce a flag to drive the tests to run in containerized mode, not sure the change would add value.

Reply:
@yuvalk please take a look.
I had to introduce another variable anyway (TESTS_IN_CONTAINER) but we don't repeat the cnf_tests_image anymore and it's probably more clear than relying on the presence of the image variable to trigger the logic.